src/common/SurgeStorage.cpp

```diff
@@ -180,18 +180,31 @@ SurgeStorage::SurgeStorage(std::string suppliedDataPath)
    if(!hasSuppliedDataPath)
    {
        const char* xdgDataPath = getenv("XDG_DATA_HOME");
+       std::string localDataPath = std::string(homePath) + "/.local/share/surge/";
        if (xdgDataPath)
+       {
            datapath = std::string(xdgDataPath) + "/Surge/";
```

Comment:
Should this be /surge?

Reply:
I'm not familiar with XDG paths. Does this path get set up based on what we have in `build-linux.sh`?

Reply:
the xdgPath is fine and is set by  freedexktop settings. I more meant "/Surge/" should be "/surge/"

Reply:
Ok, got it. I'll get that updated.